api/v1/openapi.yaml

```diff
@@ -2494,6 +2494,9 @@ definitions:
           - terminating
           - quarantined
           - maintenance
+      preferred:
```

Comment:
Looking at this description, it's not entirely clear why a backend is preferred or not. I feel like affinity related context is missing. Maybe something like affinity-preferred might be more readable (although it's not as concise)?

Reply:
Good point :100: . I have updated the description to make it clearer.